### ISSUE TITLE
Reformat code takes `use` from child module to It's parent.

### ISSUE THREAD

**Issue by mojtab23** [intellij-rust/intellij-rust#3025]
## Environment

* **Intellij-Rust plugin version: 0.2.0.2109-182**
* **Rust toolchain version: 1.30.0**
* **IDE name and version: IntelliJ IDEA 2018.2.5**
* **Operating system: Windows 10**

## Problem description
Using `Reformat Code` (Ctrl+Alt+L) takes `use` statement from sub module to It's parent module.

### Code before reformat:
``` rust
pub mod id3 {
    const TAG: &str = "TAG";
    pub mod id3v1 {
        use id3::TAG;
        pub fn parse_id3v1(file_bytes: &Vec<u8>) {
            let v1_bytes = read_id3v1_tag_bytes(&file_bytes).expect("");
            let tag = &v1_bytes[..3];
            let tag_str = str::from_utf8(tag)
                .expect("Is not valid TAG!");
            assert_eq!(tag_str, TAG, "Is not valid TAG!");
        }
    }
}
```

### After reformat:

``` rust
pub mod id3 {
    const TAG: &str = "TAG";
    use id3::TAG;
    pub mod id3v1 {
        pub fn parse_id3v1(file_bytes: &Vec<u8>) {
            let v1_bytes = read_id3v1_tag_bytes(&file_bytes).expect("");
            let tag = &v1_bytes[..3];
            let tag_str = str::from_utf8(tag)
                .expect("Is not valid TAG!");
            assert_eq!(tag_str, TAG, "Is not valid TAG!");
        }
    }
}
```

**Comment by Undin** [comment]
Duplicate of #2929
Fixed in #2969

**Comment by Undin** [comment]
@mojtab23 Check the latest plugin version, please. It should contain a fix for this
